core/src/autogluon/core/loss.py

```diff
@@ -1,5 +1,5 @@
 from mxnet.gluon import loss
-from ..core import obj
+from . import obj
 
 __all__ = ['SoftmaxCrossEntropyLoss']
 
```

Comment:
Seems like this file could be easily moved out of core to make it easier for core to become free of mxnet-dependency

Reply:
can do as follow-up PR to make this change more visible